debian/control

```diff
@@ -21,7 +21,7 @@ Breaks: libdata-alias-perl
 Depends: ${misc:Depends}, vlan, make,
  openssl, openssl-blacklist, openssl-blacklist-extra,
  mysql-server|mariadb-server|mariadb-galera-server,
- snmp, snmptrapfmt, snmp-mibs-downloader, conntrack,
+ snmp, snmptrapfmt, snmp-mibs-downloader, conntrack, wget,
```

Comment:
This is a build dependency, not a runtime dependency.
This will break packaging.

Fix it for both debian and redhat please.